optimum/onnxruntime/configuration.py

```diff
@@ -247,7 +247,8 @@ class QuantizationConfig:
         nodes_to_exclude (`list`):
             List of the nodes names to exclude when applying quantization.
         operators_to_quantize (`list`):
-            List of the operators types to quantize.
+            List of the operators types to quantize. Defaults to `["MatMul", "Add"]` but supports 
+            `['Conv','Attention', 'LSTM', 'Gather', 'Transpose', 'EmbedLayerNormalization'].`
```

Comment:
Actually there are more supported operators than these in OnnxRuntime, take a look at [this piece of code](https://github.com/microsoft/onnxruntime/blob/bdd6b00c9a6a5cdd2925762a9058e5c46f4a787f/onnxruntime/python/tools/quantization/registry.py#L37).

So I would just keep the list of default operators and remove the supported ones, maybe with a reference to this link for the interested reader.

Reply:
@regisss 
Yes you are absolutely right. 
Thanks for pointing this out. 
Will make the necessary changes.